ldap/servers/slapd/auditlog.c

```diff
@@ -194,7 +199,7 @@ log_entry_attr(Slapi_Attr *entry_attr, char *attrname, lenstr *l)
 {
     Slapi_Value **vals = attr_get_present_values(entry_attr);
     for(size_t i = 0; vals && vals[i]; i++) {
-        char log_val[256] = "";
+        char log_val[256] = {0};
```

Comment:
Why changing that ?
Setting all 255 bytes to 0 instead of only setting the first one, seems like an useless overhead (especially since it is done for each values on each attribute.)

Reply:
> Why changing that ? Setting all 255 bytes to 0 instead of only setting the first one, seems like an useless overhead (especially since it is done for each values on each attribute.)

I thought it was cleaner, but you have a good point.  I'll change it